lib/process-services-cloud/src/lib/process/process-filters/process-filters-cloud.module.ts

```diff
@@ -27,6 +27,7 @@ import { EditProcessFilterCloudComponent } from './components/edit-process-filte
 import { ProcessFilterDialogCloudComponent } from './components/process-filter-dialog-cloud.component';
 import { AppListCloudModule } from './../../app/app-list-cloud.module';
 import { DateAdapter, MAT_DATE_FORMATS } from '@angular/material/core';
+import { PeopleCloudModule } from '../../people/public-api';
```

Comment:
should be imported directly from the module file, not public-api barrel, otherwise prod mode may have issues